boards/earlgrey-nexysvideo/build.rs

```diff
@@ -1,4 +1,15 @@
+use std::env;
+
 fn main() {
     println!("cargo:rerun-if-changed=layout.ld");
     println!("cargo:rerun-if-changed=../kernel_layout.ld");
+
+    let out_dir = env::var("OUT_DIR").unwrap();
+
+    println!("cargo:rustc-link-search={}/../../../", out_dir);
+    println!("cargo:rerun-if-changed=libriscv32imc-unknown-none-elf.a");
+    println!(
+        "cargo:rustc-link-lib=static={}",
+        "riscv32imc-unknown-none-elf"
+    );
```

Comment:
am I correct in understanding that the remaining fix needed is a way to only include these lines for test builds?

Reply:
Yep